ci-operator/config/openshift/azure-storage-azcopy/openshift-azure-storage-azcopy-main.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
```

Comment:
`images`: required for image publishing (postsubmit job): https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#image-publishing-and-mirroring